qiskit/quantum_info/operators/symplectic/base_pauli.py

```diff
@@ -222,14 +222,18 @@ def commutes(self, other, qargs=None):
         b_dot_a = np.mod(np.sum(np.logical_and(z1, other._x), axis=1), 2)
         return a_dot_b == b_dot_a
 
-    def evolve(self, other, qargs=None):
+    def evolve(self, other, qargs=None, frame="h"):
```

Comment:
if it is always a boolean, then I would suggest something like `heisenberg=True/False`. Also single letter abbreviations are not good practice in general.

Reply:
We were a bit concerned that `heisenberg=False` isn't very clear at the call site as to what it is instead - Schrödinger isn't an exact opposite, since in general (though not in this case) there's more possible frames.  Enums are the best for a discrete set of choices, but I don't think strings are terrible for simple situations - Numpy uses single-letter strings, for example, in its `order` parameter (and I dunno what "A" or "K" are meant to stand for - perhaps "keep", but "A" is also a type of "keep").

The main concern with long strings though was that all three of "Schrodinger", "Schroedinger" and "Schrödinger" have downsides for English-oriented keyboards! (Respectively: incorrect spelling; non-ideal spelling; quite hard to type.)